### ISSUE TITLE
Classification of Spot-welded Joints in Laser Thermography Data using Convolutional Neural Networks. (arXiv:2010.12976v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#36]
Classification of Spot-welded Joints in Laser Thermography Data using Convolutional Neural Networks. (arXiv:2010.12976v1 [cs.CV])<br>
https://ift.tt/37IImvb<br>
<p>Spot welding is a crucial process step in various industries. However, classification of spot welding quality is still a tedious process due to the complexity and sensitivity of the test material, which drain conventional approaches to its limits. In this paper, we propose an approach for quality inspection of spot weldings using images from laser thermography data.We propose data preparation approaches based on the underlying physics of spot welded joints, heated with pulsed laser thermography by analyzing the intensity over time and derive dedicated data filters to generate training datasets. Subsequently, we utilize convolutional neural networks to classify weld quality and compare the performance of different models against each other. We achieve competitive results in terms of classifying the different welding quality classes compared to traditional approaches, reaching an accuracy of more than 95 percent. Finally, we explore the effect of different augmentation methods.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
